modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Breadcrumb.es.js

```diff
@@ -133,9 +135,9 @@ export default withRouter(({allowCreateTopicInRootTopic, history, section}) => {
 					<NewTopicModal
 						currentSectionId={section && section.id}
 						onClose={() => setVisible(false)}
-						onCreateNavigateTo={(topicName) =>
+						onCreateNavigateTo={(topic) =>
 							historyPushParser(
-								`/questions/${stringToSlug(topicName)}`
+								`/questions/${stringToSlug(topic)}`
```

Comment:
This now could be a topic name or a topic id? How stringToSlug behaves with ids? Could cause some problem?

Reply:
`stringToSlug` changes the white spaces by `-`. The function receives a string. In the string, we could have a name or an id (that is a string) but it should not be relevant for the function.

Reply:
Ok